src/Resources/Resources.Test/ScenarioTests/RoleAssignmentTests.ps1

```diff
@@ -639,6 +639,21 @@ function Test-RaGetByScope
     VerifyRoleAssignmentDeleted $newAssignment1
 }
 
+<#
```

Comment:
what is the use of this test?

Reply:
Test-RaGetOnlyByRoleDefinitionName test whether the cmdlet is able to retrieve role assignments only with the Role definition name without any additional options